### ISSUE TITLE
Is twrp a spyware?

### ISSUE THREAD

**Issue by Mweeg** [TeamWin/android_bootable_recovery#85]
- [x] I am running an official build of TWRP, downloaded from https://twrp.me/Devices/
- [x] I am running the latest version of TWRP
- [x] I have read the FAQ (https://twrp.me/FAQ/)
- [x] I have searched for my issue and it does not already exist

**Device codename**: <guamp>

#### WHAT STEPS WILL REPRODUCE THE PROBLEM?
just not be around your phone , ig.

#### WHAT IS THE EXPECTED RESULT?
twrp not being a spyware.

#### WHAT HAPPENS INSTEAD?
So basically I just saw a notification on my phone stating that TWRP has just obtained my location. (I'm talking about the app)

https://imgur.com/a/NyrvVid This is the proof you want.

**Comment by nsh07** [comment]
I have the same thought. To turn on subscription, you have to either turn on Network Statistics, or make an yearly donation. I understand that you need money to keep an open source project up and running, but what data the Network Statistics feature collects isn't right. Why would you collect my location? The network statistics page says they "provide" your "measurement data and system parameters" with their "partners", which, if I explain in simple terms, is selling your data and location to third parties.

The recovery is safe (and it's the best recovery available out there), but the app doesn't seem to be like that...

**Comment by JesusIsJehovah** [comment]
Absolutely disgusting. TWRP started spying on us, bullshit, diabolical nonsense of wicked works.
Ads, spying and all the rest, most of you are taking part in the snake oil poison and compromising and working for the Devil.

That is absolutely, if true, NOT ACCEPTABLE!!! DISGUSTING SATANIST PROGRAMMERS!!!

**Comment by JesusIsJehovah** [comment]
IF THIS IS TRUE YOU PROGRAMMERS SHOULD BE ASHAMED AND REPENT - THIS IS NOT ACCEPTABLE IF YOU ARE SPYING ON US LIKE THE REST OF THE SATANIST APPLICATIONS OUT THERE WORKING FOR THE DEVIL AND THAT ROMAN ANTICHRIST 666 COLLECTING OUR DATA.
FILTHY DISGUSTING BULLSHIT.

**Comment by JesusIsJehovah** [comment]
What versions does not spy on us? Any safe choices of version?
